README.md

```diff
@@ -4,24 +4,58 @@
 
 # Django app for City of Helsinki user infrastructure
 
-django-helusers is your friendly app for bolting authentication into Django projects for City of Helsinki. It provides the following functionalities:
+Django-helusers is your friendly app for bolting authentication into Django projects for City of Helsinki. Authentication schemes are based on [OAuth2](https://oauth.net/2/) and [OpenID Connect (OIDC)](https://openid.net/connect/).
 
-* baseline User model
+A baseline `User` model is provided that can be used with the various authentication use cases that are supported. The model supports mapping from AD groups to Django groups based on the authentication data.
+
+Additionally there are **optional** functionalities that can be used as needed.
+
+Functionalities for server needing (API) access token verification:
+
+* For servers using Django REST Framework
+* For servers not using Django REST Framework
+
+Functionalities for server needing support for OAuth2 or OIDC based authentication (e.g. into the Django admin interface):
```

Comment:
Maybe make this even more explicit:
Functionality for the normal Django login form, that allows logging in using OAuth2/OIDC in addition to username and password. This can be used fe. to allow OIDC login to Django admin.